Automattic/jetpack
# Add Google My Business site settings to sync whitelist

Follow up to https://github.com/Automattic/jetpack/pull/9322
We require those options on WPCOM side
Also see: code-D12628

#### Changes proposed in this Pull Request:

- Rename `google_my_business_keyring_id ` and `google_my_business_location_id` to `jetpack_google_my_business_keyring_id ` and `jetpack_google_my_business_location_id `
- Add `jetpack_google_my_business_keyring_id ` and `jetpack_google_my_business_location_id ` to sync whitelist so they will propogate to the shadow site

#### Testing instructions:

0. See instructions on code-D12628. You can also manually test writting and reading the settings by:
1. Install jetpack on a site and link it to WordPress.com
2. POST `{"jetpack_google_my_business_keyring_id":123456,"jetpack_google_my_business_location_id":"1234567890"}` to `https://public-api.wordpress.com/rest/v1.1/sites/{SITE_ID}/settings`
3. Assert that `get_option( 'jetpack_google_my_business_keyring_id' )` ( i.e. in wpsh ) returns the value from (2)
